### PR TITLE
Broadcast notifications

### PR DESCRIPTION
**Pull request by kgf**
Ongoing work, but smaller PRs get merged.

### CODE REVIEW

File: app/views/recipients/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1>Editing Recipient</h1>
+
+<%= render 'form' %>
+
+<%= link_to 'Show', @recipient %> |
+<%= link_to 'Back', recipients_path %>
```
Comment (david135) [inline app/views/recipients/edit.html.erb:6]: Should we use the `content_for :sidebar` approach here?

File: app/views/recipients/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<h1>New Recipient</h1>
```
Comment (david135) [inline app/views/recipients/new.html.erb:1]: Should we use the `content_for :sidebar` approach here?

File: app/views/recipients/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<p id="notice"><%= notice %></p>
```
Comment (david135) [inline app/views/recipients/show.html.erb:1]: Should we use the `content_for :sidebar` approach here?

Comment (kgf) [reply app/views/recipients/show.html.erb:1]: I agree, and I have created an issue #296

**Pull request merged**